docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this repository
+before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions
+with the project.
+
+## Commits
+
+Project is following the semantic commit messages with emoji twist, based on the
+type of work done prefix commit message with one of the following:
+  * ⭐️ for commits related to adding new features.
```

Comment:
Hey @chmelevskij,

Don't want to be old-school here but I'm not a big fan of this approach, mainly before it will slowdown writing commits (we will have to open the macOS emoji keyboard or even work http://emojipedia.org everytime we want to commit...)

I prefer the "famous" [Angular commit convention](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit) that is compliant with [standard-version](https://github.com/conventional-changelog/standard-version) for example.

Reply:
Ohh nooooo..... Just kidding. As long as there is some convention I'm happy with that, just need to agree on what it is.

Reply:
Do you have any other convention (other than Angular's one) in mind? 🧠

Reply:
Tell me and I'll update the document with our agreement.

Reply:
Not really, been using Angular one at work anyways, so that works for me.

Reply:
Okay, updated in f5e0f96